docs/custom-implementation.md

```diff
@@ -0,0 +1,45 @@
+# Custom Implementation
+
+!!! tip
+
+    Install Git, Maven and Docker Compose
+
+HIS systems, like Ozone, often need to be heavily customized based on local needs. Ozone facilitates implementer's providing custom implementation configurations via a series of tools built on Apache Maven. The starting-point for a custom implementation is the Ozone Maven Archetype which creates a skeleton project that serves as a base for any particular customizations an implementation needs.
+
+## Getting Started
+
+To get started, use Maven's archetype tools to generate a new Ozone implementation project. The command for doing this is:
+
+```bash
+ mvn archetype:generate -DarchetypeArtifactId=ozone-distro-archetype -DarchetypeGroupId=com.ozonehis
```

Comment:
The artifact id of the archetype should be `maven-archetype`

Reply:
Yeah, that got changed at some point.